tracer.spec

```diff
@@ -1,79 +1,67 @@
+%global rel 1
+
 Name:       tracer
 Version:    0.6.11
-Release:    1%{?dist}
+Release:    %{rel}%{?dist}
 Summary:    Finds outdated running applications in your system
 
 BuildArch:  noarch
-Group:      Applications/System
-License:    GPLv2
+License:    GPLv2+
 URL:        http://tracer-package.com/
-# Sources can be obtained by
-# git clone git@github.com:FrostyX/tracer.git
-# cd tracer
-# tito build --tgz
-Source0:    %{name}-%{version}.tar.gz
+Source0:    https://github.com/FrostyX/tracer/archive/%{name}-%{version}-%{rel}.tar.gz
```

Comment:
Nope. This is not the archive used in Source0. It may have the same content but does not have the same checksum.